pkg/strace/types.go

```diff
@@ -0,0 +1,14 @@
+// Copyright 2015-2018 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package strace
+
+// gvisor uses cgo and that won't do.
```

Comment:
It really doesn't -- what?

If this goes anywhere, it should be the ABI package. Or just use this one https://godoc.org/golang.org/x/sys/unix#Iovec